frontend/src/Settings/MediaManagement/MediaManagement.js

```diff
@@ -76,6 +76,24 @@ const fileDateOptions = [
     get value() {
       return translate('PhysicalReleaseDate');
     }
+  },
+  {
+    key: 'cinemasOrRelease',
```

Comment:
Wouldn't this essentially be the same as oldest in most cases? I'm struggling to see the need for all three of these. I could be missing something obvious though.

Reply:
@Qstick Is there anything left I can do for you? Or are these changes ready for approval?

Reply:
@Qstick It would be nice to have a response. I can change anything you like. It would be very kind to have this change merged.